pipeline-modules/app-service/gcp/git-access.yaml

```diff
@@ -32,19 +32,18 @@ inputs:
     - git_remote_path
     - git_login
 template: |
+  {% set workspace = '/workspace' %}   {# has to be a persistent volume #}
```

Comment:
You could probably define these vars at the beginning of the pipeline-config module. That way they should will be available to every other module afterwards.

Reply:
Well I tried that but they didn't seem to carry over...not sure why

Reply:
So actually this shouldn't be needed because this module should use the cpi-config container which has the path included..

Reply:
this is fixed now